.license_check.yaml

```diff
@@ -28,51 +28,13 @@
 # concerned, they can be considered the same.
 
 file_types:
-  - pattern: "*.service"
+  "*.service":
     type: yaml
-  - pattern: "*.spec"
+  "*.spec":
     type: yaml
-  - pattern: "*.sh"
+  "goss-testing/automated/*":
     type: shell_or_python
-  - pattern: "*.bash"
-    type: shell_or_python
-  - pattern: "*.py"
-    type: shell_or_python
-  - pattern: "Dockerfile*"
-    type: docker
-  - pattern: "*/Dockerfile*"
-    type: docker
-  - pattern: "*.java"
-    type: java
```

Comment:
Override is already deviated from base config here: `java` is actually split into `java_inline` and `java_block`.